src/handlers/metadataDelta/metadataDelta.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  ColonyActionType,
+  CreateVerifiedMemberDocument,
+  CreateVerifiedMemberMutation,
+  CreateVerifiedMemberMutationVariables,
+  GetVerifiedMemberDocument,
+  GetVerifiedMemberQuery,
+  GetVerifiedMemberQueryVariables,
+} from '~graphql';
+import { ContractEvent } from '~types';
+import { verbose, writeActionFromEvent } from '~utils';
+import {
+  isAddVerifiedMembersOperation,
+  isMetadataDeltaOperation,
+  parseOperation,
+} from './utils';
+import { mutate, query } from '~amplifyClient';
+
+export default async (event: ContractEvent): Promise<void> => {
+  const { contractAddress: colonyAddress } = event;
+  const { agent: initiatorAddress } = event.args;
+  const operationString = event.args[1];
+
+  if (!operationString) {
+    verbose('Unable to get operation for ColonyMetadataDelta event');
+  }
+
+  const operation = parseOperation(operationString);
+
+  if (!isMetadataDeltaOperation(operation)) {
+    verbose(
+      'Operation does not conform to MetadataDeltaOperation type: ',
+      operation,
+    );
+    throw new Error('Unknown operation format');
```

Comment:
As discussed, we shouldn't throw errors inside handlers (it will cause a restart loop)